test/_interface/test_rosetta_interface.py

```diff
@@ -0,0 +1,43 @@
+"""Testing enzy_htp._interface.rosetta_interface.py
+Author: Qianzhen (QZ) Shao <shaoqz@icloud.com>
+Date: 2024-02-16
+"""
+from pathlib import Path
+import re
+import pytest
+import os
+
+from enzy_htp.core.clusters.accre import Accre
+import enzy_htp.core.file_system as fs
+from enzy_htp._config.armer_config import ARMerConfig
+from enzy_htp import PDBParser
+from enzy_htp import interface, config
+
+DATA_DIR = f"{os.path.dirname(os.path.abspath(__file__))}/data/"
+WORK_DIR = f"{os.path.dirname(os.path.abspath(__file__))}/work_dir/"
+sp = PDBParser()
+ri = interface.rosetta
+
+def test_relax_w_stru():
+    """as said in the name. make sure relax works Structure() input."""
+    config._rosetta.RELAX = "$ROSETTA3/source/bin/relax.mpi.linuxgccrelease"
```

Comment:
Try using `config['rosetta.RELAX'] = /path/to/relax`

Reply:
Same for `MPI_EXECUTABLE` also

Reply:
This way it doesn't autocomplete. I would argue for using the way I did as convention.

Reply:
The reason I opted to create the `[]` operator is:
   + Runs everything through a function so there is some degree of parsing/checking for correctness. Makes it so that the error message can be more descriptive also
   + makes it easier to update settings through datafiles like `~/.eh_config` or in workflow scripts

I am not opposed to supporting direct access, but would prefer to do it where there is not underscore since that looks weird and is generally not a good process. The two options are: 
   + provide support for the `__getattr__` so that when you call `config.rosetta` it is actually `config['rosetta']` **this is preferred IMO**
   + change all attributes from `_package` to `package`, (i.e. config._rosetta becomes config.rosetta); **this is less prefferred**

It may make the most sense to just punt this to an issue for later. I guess it is alright to make the convention to not use the brackets, but they are very nice for scripting/setting global configurations

Reply:
Cool! I will apply it now.

Reply:
I made them properties so that autocomplete works. The __getattr__ still can't work with autocomplete, which can't process code but can only read type hinting.

Reply:
Cool! As long as we keep both I am ok with it.